cloudify_hostpool/storage/base.py

```diff
@@ -58,8 +58,8 @@ def update_server(self, global_id, server):
         :param global_id: The global id of the server.
         :param server: The server dictionary containing the new values.
 
-        :return False if no change occurred, True otherwise.
-        :rtype bool
+        :return (False, None) if no change occurred, (True, server) otherwise.
```

Comment:
is the tuple necessary? maybe we can skip the boolean...you can still do a conditional on the result
None is equivelant to False and server to True.